vlib/build/README.md

```diff
@@ -0,0 +1,32 @@
+## Description
+
+`build` provides a small build system leveraging V(SH) for the buildscript.
+
+## Example
+
+```v
+#!/usr/bin/env v
```

Comment:
Run `VAUTOFIX=1 v check-md vlib/build/README.md` 2 times to fix the formatting issue.